### PR TITLE
Handle 503 errors and throw an exception on failure

### PR DESCRIPTION
**Pull request by AkeemMcLennon**

### CODE REVIEW

File: b2_storage/backblaze_b2.py [hunk 1]
```diff
@@ -55,11 +57,14 @@ def upload_file(self, name, content):
         }
 
         download_response = requests.post(url, headers=headers, data=content.read())
+        # Status is 503: Service unavailable. Try again
```
Comment (AkeemMcLennon) [inline b2_storage/backblaze_b2.py:60]: It looks like sometimes servers go down (maybe related to DNS issues lately?) but the next request seems to route to a new server

**Review by royendgel — CHANGES_REQUESTED** [review]
Great! I
can you change line 65 into attempts += 1  ?

**Comment by AkeemMcLennon** [comment]
Sorry for the late reply. This has been updated

**Comment by royendgel** [comment]
Merged !

**Pull request merged**